alicloud/data_source_alicloud_security_groups.go

```diff
@@ -0,0 +1,168 @@
+package alicloud
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/denverdino/aliyungo/ecs"
+	"github.com/denverdino/aliyungo/util"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type SecurityGroup struct {
+	Attribute    ecs.DescribeSecurityGroupAttributeResponse
+	CreationTime util.ISO6801Time
```

Comment:
I think type SecurityGroup is needless and ecs.DescribeSecurityGroupAttributeResponse is ok.

Reply:
Hi @malferov Do you think it need a new type 'SecurityGroup' to support the next function?

Reply:
In addition, please add a docs for security groups datasource in folder terraform-provider-alicloud/website/docs/d/ . Thanks a lot.

Reply:
New type is needless indeed. Please find update.

Reply:
Docs have been added.